src/main/java/com/jherrick/BasicBoardEvaluator.java

```diff
@@ -1,28 +1,90 @@
 package com.jherrick;
 
 public class BasicBoardEvaluator implements BoardEvaluator{
+    
+    // =============== Instance variables ===============
+    
+    // Evaluation metrics. Only add or remove metrics, do not hard code in. Should be hard coded in EVAL_CONSTANTS
+    private static int c0 = Constants.EVAL_CONSTANTS[0];
+	private static int c1 = Constants.EVAL_CONSTANTS[1];
+	// private static int c2 = Constants.EVAL_CONSTANTS[2];
```

Comment:
Small comment but don't leave commented-out code in source control. Since git is versioned, you can delete it and we can always go back and get it if we need it.

Reply:
Also make the indentation consistent for c0 and c1